lambda/lambda-import/README.md

```diff
@@ -0,0 +1,49 @@
+# MetaKeep Lambda - Import UniswapV2Router and Swap Matic tokens for USDC
+
```

Comment:
The folder name should be something like `token-swap`

Reply:
done